packages/e2e-tests/helpers/index.ts

```diff
@@ -22,21 +22,37 @@ export async function openDevToolsTab(page: Page) {
   await header.waitFor();
 
   // The DevTools tab won't exist if it's a Node recording.
-  const tab = page.locator('[data-test-id="ViewToggle-DevTools"]');
+  const tab = getDevToolsTab(page);
   if (await tab.isVisible()) {
     await tab.click();
   }
 }
 
+export const getDevToolsTab = (page: Page) => page.locator('[data-test-id="ViewToggle-DevTools"]');
+
+export const getViewerTab = (page: Page) => page.locator('[data-test-id="ViewToggle-Viewer"]');
+
 export async function openViewerTab(page: Page) {
   // Wait for the page to load enough that it's safe to continue.
   const header = page.locator('[data-test-name="Header"]');
   await header.waitFor();
 
-  const tab = page.locator('[data-test-id="ViewToggle-Viewer"]');
+  const tab = getViewerTab(page);
   await tab.click();
 }
 
+export async function isViewerTabActive(page: Page) {
+  const tab = getViewerTab(page);
+  const classes = await getElementClasses(tab);
+  return classes.includes("active");
+}
+
+export async function isDevToolsTabActive(page: Page) {
+  const tab = getDevToolsTab(page);
+  const classes = await getElementClasses(tab);
+  return classes.includes("active");
+}
```

Comment:
Seems like these are working so 👍🏼 but if they ever appear flaky, we might have good results from migrating them to the "wait for" pattern (`await waitFor(async () => assertThatMightThrow())`